jaxrs-spec/src/main/asciidoc/chapters/introduction/_status.adoc

```diff
@@ -20,11 +20,11 @@ The corresponding Javadocs can be found online at:
 
 https://jakarta.ee/specifications/restful-ws/3.0/apidocs/
 
-The reference implementation can be obtained from:
+A compatible implementation can be obtained from:
```

Comment:
Maybe we should omit this, unless we are fair and list *all* compatible implementations?

Reply:
I would be fine with both.